frontend/src/scenes/surveys/SurveyTemplates.tsx

```diff
@@ -44,10 +44,10 @@ export function SurveyTemplates(): JSX.Element {
                                 reportSurveyTemplateClicked(template.type)
                             }}
                         >
-                            <span className="mb-2 text-md">
+                            <span className="text-md">
                                 <b>{template.type}</b>
                             </span>
-                            <span className="flex flex-wrap text-xs text-muted max-w-80 font-medium mb-2">
+                            <span className="flex flex-wrap text-xs text-muted max-w-80 font-medium mb-3">
```

Comment:
Since we're removing the margin bottom in the title, I think it's fine to leave this part as is, looks better to me IMO

```suggestion
                            <span className="flex flex-wrap text-xs text-muted max-w-80 font-medium mb-2">
```

Reply:
Ehh actually I don't really care looks fine either way